services/app-web/src/components/HealthPlanPackageTable/HealthPlanPackageTable.tsx

```diff
@@ -137,13 +148,41 @@ const columnHash = atomWithHash('filters', [] as ColumnFiltersState, {
     deserialize: fromReadableUrlToColumnFilters,
 })
 
+/* transform react-table's ColumnFilterState (stringified, formatted, and stored in the URL) to react-select's FilterOptionType
+    and return only the items matching the FilterSelect component that's calling the function*/
+const getSelectedFiltersFromUrl = (
+    columnFilters: ColumnFiltersState,
+    id: string
+) => {
+    type TempRecord = { value: string; label: string; id: string }
+    const valuesFromUrl = [] as TempRecord[]
```

Comment:
can you not write `const valuesFromUrl: TempRecord[] = []` and avoid the "as"?

Reply:
Oh I didn't realize this was just getting moved from somewhere else. Don't sweat it